src/resolvers/resolverUtils.ts

```diff
@@ -179,6 +179,18 @@ async function getSchedule(year: number): Promise<Schedule | null> {
             startTime: meetingTime.startTime,
           }));
         }),
+        professors: [
+          {
+            id: course.user!.id,
+            username: course.user!.username,
+            password: course.user!.password,
+            displayName: course.user!.displayName,
+            role: course.user!.role as Role,
+            preferences: prismaPrefsToGraphQLPrefs(course.user!.preference),
+            active: course.user!.active,
+            hasPeng: course.user!.hasPeng,
```

Comment:
I think this is a bit risky to do...
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator
I'd rather we handle it with some default values instead.